src/main/java/com/google/sps/servlets/SentimentServlet.java

```diff
@@ -94,4 +95,8 @@ private Float calculateSentimentScore(String text) {
 
     return score;
   }
+
+  public LanguageServiceClient createLanguageServiceClient() throws IOException {
+    return LanguageServiceClient.create();
```

Comment:
Any strong reason to make this a separate method instead of calling LSC.create directly in calculateSentimentScore?

Reply:
Calling LSC.create() directly in the method would replace the mock of LSC that I need for testing, this is the work around Juan and I came up with in our 1:1 yesterday :/ By using this helper method we can stub it to return the mock.

I also tried using PowerMockito to stub create() for any instance of LSC but wasn't able to get it to work. Might try it again later today.